src/main/java/com/minecolonies/coremod/colony/buildings/AbstractCitizenAssignable.java

```diff
@@ -132,7 +132,6 @@ public void removeCitizen(final CitizenData citizen)
         if (isCitizenAssigned(citizen))
         {
             assignedCitizen.remove(citizen);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This call to "remove()" may be a performance hot spot if the collection is large. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2250)